src/facet.ts

```diff
@@ -1,4 +1,7 @@
-import {FacetFieldDef} from './fielddef';
+import {FieldDef} from './fielddef';
+
+// TODO: add more facet properties
+export type FacetFieldDef = FieldDef;
```

Comment:
The type has to be ordinal, nominal, or quantitative with bin, or time with timeunit. Should we enforce that?